### PR TITLE
Task 259 - Sticky Navigation

### LINKED ISSUE #259: Related Links - Sticky left navigation
Sticky left navigation to list related links on guide detail page and individual chapter page.

The Related Links block is a list of links that the author will manage for each page.  The author will edit the block on the page and add a title and URL link.

### PR DESCRIPTION
**Pull request by amlan18041996**
Fix #259 

Test URLs:
- Before: https://main--danaher-abcam--aemsites.hlx.live/en-us/technical-resources/guides/elisa-guide
- After: https://259-sticky-navigation--danaher-abcam--aemsites.hlx.live/en-us/technical-resources/guides/elisa-guide

### CODE REVIEW

**Review by sharanagoudapatil — COMMENTED** [review]
image

- Please rename block name from 'sticky-links' to 'related-links' (just for the meaningful name). 
- Replace 'Sticky Links' with 'Related Links' in model definitions as well.
- Fix lint errors also as part of the code/build.

**Comment by amlan18041996** [comment]
> image
> 
> * Please rename block name from 'sticky-links' to 'related-links' (just for the meaningful name).
> * Replace 'Sticky Links' with 'Related Links' in model definitions as well.
> * Fix lint errors also as part of the code/build.

Kindly check it again as I made the changes which are listed here.

File: eds/blocks/related-links/_related-links.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (rgravitvl) [inline eds/blocks/related-links/_related-links.json:1]: Are we still using related-links??

File: eds/templates/guides/guides.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { buildBlock, getMetadata } from '../../scripts/aem.js';
```
Comment (rgravitvl) [inline eds/templates/guides/guides.js:1]: there is already guide template, please check

**Review by rgravitvl — CHANGES_REQUESTED** [review]

**Review by rgravitvl — APPROVED** [review]

**Review by pardeepgera23 — APPROVED** [review]

**Review by shivangi1422 — APPROVED** [review]

**Pull request merged**